radix_sort/radix_sort.py

```diff
@@ -0,0 +1,53 @@
+# Following is the implementation of Radix Sort.
+
+
+def radix_sort(arr, radix=10):
+    """
+    :param arr: Iterable of elements to sort.
+    :param radix: Base of input numbers
+    :return: Sorted list of input.
+    Time complexity: O(d * (n + b))
+    where, n is the size of input list.
+           b is base of representation
+           d is number of digits in largest number
+    Space complexity: O(n + k)
+    where, k is the range of input.
+    """
+    max_length = False
+    tmp, digit = -1, 1
+
+    while not max_length:
+        max_length = True
+        # declare and initialize buckets
+        buckets = [[] for _ in range(radix)]
+
```

Comment:
I separated declarations and main algo with this line, making it a little more readable. I think its good to separate major things via blank lines. your views?